### PR TITLE
delete winter brake notice

### PR DESCRIPTION
**Pull request by alax99**
Another one

### CODE REVIEW

**Comment by zixuan75** [comment]
Wait for review...

File: contributing.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 # Welcome to The React Project!
```
Comment (zixuan75) [inline contributing.md]: Yeah, Winter brake is almost done.

Comment (zixuan75) [reply contributing.md]: True

Comment (alax99) [reply contributing.md]: Yeah, Team says it's done now,.

**Comment by alax99** [comment]
Just running on paste, wait

**Comment by alax99** [comment]
Back now.

**Comment by zixuan75** [comment]
Uh, well nothing

**Comment by alax99** [comment]
Not huh?

**Comment by alax99** [comment]
Just wifi

**Comment by zixuan75** [comment]
HUh?

**Comment by zixuan75** [comment]
Sh. s.h, sh

**Comment by alax99** [comment]
Ok.

**Comment by zixuan75** [comment]
Hmm..

**Comment by zixuan75** [comment]
Hey, done now

**Pull request merged**